packages/venia-concept/src/components/Checkout/form.css

```
@@ -9,6 +9,15 @@
     composes: footer from './flow.css';
 }
 
+.informationPrompt {
+    color: orangered;
```

```suggestion
    color: rgb(var(--venia-orange));
```

You should define this color as a custom property called `--venia-orange` in `src/index.css`, and set its value to `241, 99, 33`.